org.mume.MMapper.json

```diff
@@ -0,0 +1,167 @@
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/arximboldi/immer.git",
+          "tag": "v0.8.1"
+        }
+      ],
+      "build-commands": [
+        "cp -r immer /app/include/"
+      ]
+    },
+    {
+      "name": "libsecret",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dmanpage=false",
+        "-Dvapi=false",
+        "-Dgtk_doc=false",
+        "-Dintrospection=false"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.4.tar.xz",
+          "sha256": "163d08d783be6d4ab9a979ceb5a4fecbc1d9660d3c34168c581301cd53912b20",
+          "x-checker-data": {
+            "type": "gnome",
+            "name": "libsecret",
+            "stable-only": true
+          }
+        }
+      ]
+    },
+    {
+      "name": "qtkeychain",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/frankosterfeld/qtkeychain/archive/0.14.3.tar.gz",
+          "sha256": "a22c708f351431d8736a0ac5c562414f2b7bb919a6292cbca1ff7ac0849cb0a7",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 4138,
+            "stable-only": true,
+            "url-template": "https://github.com/frankosterfeld/qtkeychain/archive/$version.tar.gz"
+          }
+        }
+      ],
+      "config-opts": [
+        "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+        "-DLIB_INSTALL_DIR=/app/lib",
+        "-DBUILD_TRANSLATIONS=OFF",
+        "-DBUILD_WITH_QT6=ON",
+        "-DCMAKE_PREFIX_PATH=/app"
+      ]
+    },
+    {
+      "name": "map",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/MUME/arda.git",
+          "tag": "42"
+        }
+      ],
+      "build-commands": [
+        "cp arda.xml /app/"
+      ]
+    },
+    {
+      "name": "images",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/MUME/images.git",
+          "tag": "main"
+        }
+      ],
+      "build-commands": [
+        "mkdir -p /app/images",
+        "cp -r . /app/images/"
+      ]
+    },
+    {
+      "name": "mmapper",
+      "buildsystem": "cmake-ninja",
+      "builddir": true,
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/MUME/MMapper.git",
+          "tag": "beta"
+        }
+      ],
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DWITH_WEBSOCKET=ON",
+        "-DWITH_QTKEYCHAIN=ON",
+        "-DWITH_MAP=/app/arda.xml",
+        "-DWITH_IMAGES=/app/images/",
+        "-DWITH_TESTS=OFF",
+        "-DOPENSSL_ROOT_DIR=/app",
+        "-DCMAKE_PREFIX_PATH=/app/lib/qt6:/app"
+      ],
+      "build-commands": [
+        "sed -i 's|^Exec=.*|Exec=/app/bin/mmapper|' ../src/org.mume.MMapper.desktop"
```

Comment:
never use `sed`. There is the `desktop-file-edit` command. Also exec should NOT be an absolute pass. It should match `command` in the manifest more or less the substitutions allowed in desktop files.